src/main/java/com/google/sps/servlets/AllMarkersServlet.java

```diff
@@ -49,12 +49,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Query query = new Query("Recommendation");
     query.addProjection(new PropertyProjection("latitude", Double.class));
     query.addProjection(new PropertyProjection("longitude", Double.class));
+    query.addProjection(new PropertyProjection("category", String.class));
     PreparedQuery results = DatastoreServiceFactory.getDatastoreService().prepare(query);
     for (Entity entity : results.asIterable()) {
       double lat = (double) entity.getProperty("latitude");
       double lng = (double) entity.getProperty("longitude");
+      String category = entity.getProperty("category").toString();
```

Comment:
```suggestion
      String category = (String) entity.getProperty("category");
```
nit nit: maybe to make it the same with the other getProperty for String, but this is perfectly fine

Reply:
Good spot, thanks Rania! The [toString method](https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#toString()) on any object returns a string representation of that object in this format `getClass().getName() + '@' + Integer.toHexString(hashCode())`

This is [overridden in the String class](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#toString()) and it will return the string itself instead.

I agree with making it consistent with other properties though and to avoid any potential errors (e.g. having the `category` stored as any other object than a String)

Reply:
Thanks! Good job for spotting that!